packages/react/src/progress-indicator/ProgressIndicator.tsx

```diff
@@ -56,6 +56,12 @@ export const ProgressIndicator = ({
 	 */
 	const hasExplicitActiveDeclared = items.some(({ isActive }) => isActive);
 
+	if (process.env.NODE_ENV !== 'production' && hasExplicitActiveDeclared) {
+		console.warn(
+			'ProgressIndicator: The `isActive` prop is deprecated. Please use `activePath` in the parent component, `<ProgressIndicator />`.'
```

Comment:
?
```suggestion
			'ProgressIndicator: The `isActive` key is deprecated. Please use `activePath` in the parent component, `<ProgressIndicator />`.'
```

Reply:
Oh yes, that will make more sense to a consumer. Nice one.